### PR TITLE
Add style attributes to table td and th elements

### PR DESCRIPTION
**Pull request by nlisgo**

### CODE REVIEW

File: lib/xsl/jats-to-html.xsl [hunk 1]
```diff
@@ -1105,6 +1105,11 @@
                     <xsl:value-of select="concat('table-', @align)"/>
                 </xsl:attribute>
             </xsl:if>
+            <xsl:if test="@style">
+                <xsl:attribute name="class">
```
Comment (davidcmoulton) [inline lib/xsl/jats-to-html.xsl]: Should be `<xsl:attribute name="style">` if we're letting the styles come through as it.

Comment (nlisgo) [inline lib/xsl/jats-to-html.xsl]: Good spot. Silly mistake.

**Comment by davidcmoulton** [comment]
lgtm.

**Comment by davidcmoulton** [comment]
lgtm.

**Pull request merged**